src/manifest/MsQuicEtw.man

```diff
@@ -2650,6 +2689,24 @@
               template="tid_CONN"
               value="5179"
               />
+          <event
+              keywords="ut:Connection ut:LowVolume"
+              level="win:Informational"
```

Comment:
I'm thinking error level instead. Thoughts? (Same for ALPN one below)

Reply:
I was matching the existing level for the log which says transport shutdown reason. But I think this qualifies as error level.